src/app.jsx

```diff
@@ -42,15 +39,19 @@ class App extends React.Component {
   }
 
   render() {
-    return (
```

Comment:
this is awesome! probably gonna fix most if not all of our downstream request errors